internal/xdscache/v3/listener.go

```diff
@@ -481,8 +481,8 @@ func (c *ListenerCache) OnChange(root *dag.DAG) {
 					Codec(envoy_v3.CodecForVersions(cfg.DefaultHTTPVersions...)).
 					AddFilter(envoy_v3.FilterMisdirectedRequests(vh.VirtualHost.Name)).
 					DefaultFilters().
-					AddFilter(authFilter).
-					AddFilter(envoy_v3.FilterJWTAuth(vh.JWTProviders)).
+					AddFilter(envoy_v3.FilterJWTAuthN(vh.JWTProviders)).
```

Comment:
Ideally we can add a test case for this. I won't block the PR on it, but it would be very nice to have.